src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -108,12 +101,8 @@ import os
 def main(context):
     # Why not try the Appwrite SDK?
     #
-    # client = (
-    #     Client()
-    #         .set_endpoint("https://cloud.appwrite.io/v1")
-    #         .set_project(os.environ["APPWRITE_FUNCTION_PROJECT_ID"])
-    #         .set_key(os.environ["APPWRITE_API_KEY"])
-    # )
+    # client = Client()
+    # client.set_key(context.req.headers.get('x-appwrite-key')) Set API key; it's automatically provided by Appwrite
```

Comment:
Let's keep the multiline like before so we can make it clear the comment is for the `setKey()` part.